src/main/java/com/fasterxml/jackson/databind/cfg/MapperConfig.java

```diff
@@ -408,6 +408,42 @@ public final BeanDescription introspectDirectClassAnnotations(JavaType type) {
     }
 
     /**
+     * Accessor for default property inclusion to use for serialization,
+     * considering possible per-type override for given base type and
+     * possible per-type override for given property type.<br>
+     * NOTE: if no override found, defaults to value returned by
+     * {@link #getDefaultPropertyInclusion()}.
+     *
+     * @since 2.9
+     */
+    public abstract JsonInclude.Value getDefaultInclusion(Class<?> baseType,
+            Class<?> propertyType);
+
+    /**
+     * Accessor for default property inclusion to use for serialization,
+     * considering possible per-type override for given base type and
+     * possible per-type override for given property type; but
+     * if none found, returning given <code>defaultIncl</code>
+     *
+     * @param defaultIncl Inclusion setting to return if no overrides found.
+     *
+     * @since 2.9
+     */
+    public JsonInclude.Value getDefaultInclusion(Class<?> baseType,
+            Class<?> propertyType, JsonInclude.Value defaultIncl)
+    {
+        JsonInclude.Value v0 = getConfigOverride(propertyType).getIncludeAsProperty();
+        if (v0 != null) {
+            return v0;
```

Comment:
May not be a big issue, but usually it'd make sense to merge with defaults;

    JsonInclude.Value.merge(defaultIncl, v0);

method does handle `null` without problems (in case `defaultIncl` passed as `null`).

I think this mostly matters for things like global defaults for just one aspects (like don't include `null`s for values); not sure if this is commonly used (probably isn't), but easy to add.